eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -81,6 +83,123 @@ spec =
+        assertNoCommittee
+      it "SPO needs to vote on security-relevant parameter changes" $ do
+        (drep, ccCred) <- electBasicCommittee
+        khPool <- setupPoolWithStake $ Coin 42_000_000
+        gaidThreshold <- do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppPoolVotingThresholdsL . pvtSecurityRelevantL) `shouldBe` minBound
+          rew <- registerRewardAccount
+          let ppUpdate =
+                emptyPParamsUpdate
+                  & ppuPoolVotingThresholdsL
+                    .~ SJust
+                      PoolVotingThresholds
+                        { pvtSecurityRelevant = 1 %! 2
+                        , pvtMotionNoConfidence = 1 %! 2
+                        , pvtHardForkInitiation = 1 %! 2
+                        , pvtCommitteeNormal = 1 %! 2
+                        , pvtCommitteeNoConfidence = 1 %! 2
+                        }
+                  & ppuGovActionLifetimeL .~ SJust (EpochInterval 100)
+          gaidThreshold <-
+            submitProposal $
+              ProposalProcedure
+                { pProcReturnAddr = rew
+                , pProcGovAction = ParameterChange SNothing ppUpdate SNothing
+                , pProcDeposit = pp ^. ppGovActionDepositL
+                , pProcAnchor = def
+                }
+          submitYesVote_ (DRepVoter drep) gaidThreshold
+          submitYesVote_ (CommitteeVoter ccCred) gaidThreshold
+          logAcceptedRatio gaidThreshold
+          pure gaidThreshold
+        passEpoch
+        logAcceptedRatio gaidThreshold
+        passEpoch
+        gaidMinFee <- do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppPoolVotingThresholdsL . pvtSecurityRelevantL) `shouldBe` (1 %! 2)
+          rew <- registerRewardAccount
+          gaidMinFee <-
+            submitProposal $
+              ProposalProcedure
+                { pProcReturnAddr = rew
+                , pProcGovAction =
+                    ParameterChange
+                      (SJust (PrevGovActionId gaidThreshold))
+                      ( emptyPParamsUpdate
+                          & ppuMinFeeAL .~ SJust (Coin 12_345)
+                      )
+                      SNothing
+                , pProcDeposit = pp ^. ppGovActionDepositL
+                , pProcAnchor = def
+                }
+          submitYesVote_ (DRepVoter drep) gaidMinFee
+          submitYesVote_ (CommitteeVoter ccCred) gaidMinFee
+          pure gaidMinFee
+        passEpoch
+        logAcceptedRatio gaidMinFee
+        passEpoch
+        do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppMinFeeAL) `shouldBe` Coin 44
+          submitYesVote_ (StakePoolVoter khPool) gaidMinFee
+        passEpoch
+        logStakeDistr
+        logAcceptedRatio gaidMinFee
+        logRatificationChecks gaidMinFee
+        passEpoch
+        pp <- getsNES $ nesEsL . curPParamsEpochStateL
+        (pp ^. ppMinFeeAL) `shouldBe` Coin 12_345
```

Comment:
Same here. Use a binding for `newMinFeeA` instead at the beginning of the test for both update and verification
```suggestion
        (pp ^. ppMinFeeAL) `shouldBe` newMinFeeA
```